cmd/olympus/actions/app.go

```diff
@@ -62,7 +63,14 @@ func App(config *AppConfig) (*buffalo.App, error) {
 		return nil, err
 	}
 
-	lggr := log.New(env.CloudRuntime(), env.LogLevel())
+	lvl, err := env.LogLevel()
+	if err != nil {
+		return nil, err
+	}
+	lggr := log.New(env.CloudRuntime(), lvl)
+	if ENV == "production" {
+		lvl = logrus.PanicLevel
```

Comment:
could we have a way to override this level for production? For example, if you want to deploy a canary of a new release, you may want to deploy to a single node/pod/process with verbose logging and see what buffalo logs say when you turn on a small bit of traffic